Geometry_Engine/Query/LineIntersection.cs

```diff
@@ -482,6 +482,76 @@ public static List<Point> LineIntersections(this Polyline curve1, Polyline curve
             return iPts.CullDuplicates(tolerance);
         }
 
+        /***************************************************/
+
+        [Description("Find segments of a line lying inside the region defined by a polyline.")]
+        [Input("line", "A line to check for intersections with a region defined by the input polylines.")]
+        [Input("pLine", "A polyline defining a closed region that the input line potentially intersects.")]
+        [Input("tolerance", "Minimum length required of new intersection lines.")]
+        [Output("intersections", "Segments of the input line that lie inside the region defined by the input polyline.")]
+        public static List<Line> LineIntersections(this Line line, Polyline pLine, double tolerance = Tolerance.Distance)
+        {
+            var intersections = new List<Line>();
```

Comment:
```suggestion
            List<Line> intersections = new List<Line>();
```

Very well aware that the `var` will do the same, but in more or less all other places in the Engine we have declared explicit types. Would keep on doing that in this method.

Same for all other variable declarations in this and other added Geometry_Engine methods